File: src/main/java/co/glebmavi/webproglab4/dto/HitResponse.java
package co.glebmavi.webproglab4.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.Date;

public record HitResponse(
        @JsonProperty("id") long id,
        String owner,
        double x,
        double y,
        double r,
        boolean hit,
        Date currDate,
        long execTime) {
}

File: src/main/webapp/AngularClient/src/app/model/HitResponse.ts
import {HitRequest} from "./HitRequest";

export class HitResponse {
  id : number;
  currDate : Date;
  execTime : number;
  request : HitRequest;
  hit : boolean;
  constructor() {
    this.id = 0;
    this.currDate = new Date();
    this.execTime = 0;
    this.request = new HitRequest();
    this.hit = false;
  }
}
